lib/features/about/card.widget.dart

```diff
@@ -22,14 +23,33 @@ class AboutCard extends StatelessWidget {
           ResponsiveRowColumn(
             rowMainAxisAlignment: MainAxisAlignment.spaceAround,
             layout: isColumnLayout ? ResponsiveRowColumnType.COLUMN : ResponsiveRowColumnType.ROW,
-            children: const [
-              ResponsiveRowColumnItem(
+            children: [
+              const ResponsiveRowColumnItem(
                 rowFlex: 1,
                 child: CardHeaderWidget(),
               ),
               ResponsiveRowColumnItem(
                 rowFlex: 1,
-                child: AppVersionTable(),
+                child: ResponsiveRowColumn(
+                  rowMainAxisAlignment: MainAxisAlignment.spaceAround,
+                  layout: ResponsiveRowColumnType.COLUMN,
+                  children: [
+                    ResponsiveRowColumnItem(
+                      rowFlex: 1,
+                      child: Padding(
+                          padding: EdgeInsets.fromLTRB(spacing(1), spacing(3), spacing(1), spacing(1.25)),
+                          child: const AppVersionTable(),
+                      ),
+                    ),
+                    ResponsiveRowColumnItem(
+                      rowFlex: 1,
+                      child: Padding(
+                        padding: EdgeInsets.fromLTRB(spacing(1), spacing(3), spacing(1), spacing(1.25)),
+                        child: const AuthorInfoTable(),
+                      ),
+                    ),
+                  ],
+                ),
```

Comment:
❤️  the final result

However I'm wondering if we can simplify 🤔 with a `Column` because I dont see the advantage of responsiveness widgets (`ResponsiveRowColumn` & `ResponsiveRowColumnItem`) since it has been addressed by the wrapping code

Let me know what you think!

For the padding, each element (`Column`, `AppVersionTable`, `AuthorInfoTable`) can be wrapped independently inside a `Padding` parent component, or sometimes a simple `SizedBox` is easier to read inside a `Column` so you can try to play with these options like so:

```suggestion
                child: Column(
                  children: const [
                    SizedBox(height: spacing(1))
                    AppVersionTable(),
                    SizedBox(height: spacing(1))
                    AuthorInfoTable(),
                  ],
```

Reply:
Yes, you are absolutely right.

A `Column` works just as great, and is also a lot easier to read :+1: